capitulo4.md

```diff
@@ -0,0 +1,190 @@
+|---------|----------------|
+| `\\`     | Barra invertida (\)           |
+| `\'`     | Comilla simple (')          |
+| `\"`     | Comilla doble (") |
+| `\n`     | Salto de línea       |
+| `\t`     | Tabulador       |
+
+Existen más caracteres de escape en Python, pero estos son los caracteres que más probablemente va a necesitar durante la creación de juegos.
+
+## Comillas Simples y Dobles
+Mientras todavía estamos en una terminal interactiva, veamos un poco más a fondo a las comillas. Las cadenas no siempre tienen que estar entre comillas simples en Python. También se pueden poner entre comillas dobles. Estas dos líneas a continuación van a imprimir la misma cosa:
+<pre>
+>>> print('Hola Mundo')
+Hola Mundo
+>>> print("Hola Mundo")
+Hola Mundo
+</pre>
+
+Pero no se pueden mezclar las comillas simples con las dobles. Esta siguiente línea va a retornar un error porque utiliza ambas comillas al mismo tiempo:
+
+<pre>
+>>> print('Hola mundo")
+SyntaxError: EOL while scanning single-quoted string
+</pre>
+
+Personalmente prefiero utilizar comillas simples para no tener que mantener presionada la tecla SHIFT para escribirlas. Son mucho más fáciles de escribir y a Python no le importa cuál de las dos utilicemos.
+
+Tenga en cuenta que al igual que necesita la barra invertida (`\'`), para tener una comilla simple en una cadena encerrada entre comillas simples, también va a necesitar la barra invertida (`\"`) para poder tener comillas dobles en una cadena encerrada por comillas dobles. Veamos los ejemplos a continuación:
+
+<pre>
+>>> print('Le pedí prestado el carro a Pedro pa\'ir al pueblo. Él dijo, "Seguro."')
+Le pedí prestado el carro a Pedro pa'ir al pueblo. Él dijo, "Seguro."
+>>> print("Él dijo, \"No puedo creer que lo dejaste llevarse el carro pa'l pueblo\"")
+Él dijo, "No puedo creer que lo dejaste llevarse el carro pa'l pueblo"
+</pre>
+
+En las cadenas de comillas simples no necesitas escapar las comillas dobles, y en las cadenas de comillas dobles no necesitas escapar las comillas simples. El intérprete de Python tiene inteligencia suficiente para saber que si una cadena comienza con un tipo de comillas, el otro tipo de comillas no significa que la cadena está terminada.
+
+## El parámetro de palabra clave `end` de la función print()
+
+Ahora regresemos al archivo chistes.py y observemos las líneas de la 9 a las 6:
+
+<pre>
+ 9. print('¿En qué se parecen una familia, un bombero y un barco?')
+10. input()
+11. print("No sé... ¿en qué se parecen?")
+12. input()
+13. print('En que el bombero y el barco tienen casco.')
+14. input()
+15. print('¿Y la familia?', end='')
+16. print(' -Bien, gracias.')
+</pre>
+
+¿Te diste cuenta del segundo parámetro en el print de la línea 15?. Normalmente, `print()` añade un salto de línea al final de la cadena que imprime. Por esta razón, una función `print()` en blanco tan solo imprimirá una nueva línea. Pero la función `print()` tiene la opción de un segundo parámetro (que tiene nombre “end” (fin)).
```

Comment:
Te diste cuenta del -> Notó el
en el print -> en la función `print()`
un salto de -> una nueva
cadena -> cadena de caracteres
tiene la opción de un segundo parámetro: `end` (fin en inglés).